services/blockchain-app-registry/shared/utils/downloadRepository.js

```diff
@@ -127,33 +131,30 @@ const getDiff = async (lastSyncedCommitHash, latestCommitHash) => {
 	} catch (error) {
 		let errorMsg = error.message;
 		if (Array.isArray(error)) errorMsg = error.map(e => e.message).join('\n');
-		logger.error(`Unable to get diff due to: ${errorMsg}`);
+		logger.warn(`URL invocation failed: ${url}.`);
+		logger.error(`Unable to get diff due to: ${errorMsg}.`);
 		throw error;
 	}
 };
 
 const filterMetaConfigFilesByNetwork = async (network, filesChanged) => {
```

Comment:
As a practice, please assign default values in the method params.
```suggestion
const filterMetaConfigFilesByNetwork = async (network, filesChanged = []) => {
```

Reply:
Applied.